src/game/client/draw/w3dtruckdraw.cpp

```diff
@@ -0,0 +1,631 @@
+                                        if (angle < angle2d) {
+                                            angle = angle2d;
+                                        }
+
+                                        if (angle > 0.0f) {
+                                            angle = 0.0f;
+                                        }
+                                    }
+                                }
+
+                                m_cabRotation += (angle - m_cabRotation) * data->m_rotationDamping;
+                                m.Rotate_Z(m_cabRotation);
+                                Get_Render_Object()->Capture_Bone(m_cabBone);
+                                Get_Render_Object()->Control_Bone(m_cabBone, m);
+
+                                if (info != nullptr && m_trailerBone != 0) {
+                                    angle = -info->m_wheelAngle * data->m_trailerRotationMultiplier;
+                                    m_trailerRotation += (angle - m_trailerRotation) * data->m_rotationDamping;
+                                    m.Make_Identity();
+                                    m.Rotate_Z(m_trailerRotation);
+                                    Get_Render_Object()->Capture_Bone(m_trailerBone);
+                                    Get_Render_Object()->Control_Bone(m_trailerBone, m);
+                                }
+                            }
+
+                            if (m_frontLeftTireBone != 0 || m_rearLeftTireBone != 0) {
+                                float addition = data->m_powerslideRotationAddition;
+
+                                if (update != nullptr) {
+                                    const Locomotor *locomotor = update->Get_Locomotor();
+
+                                    if (locomotor != nullptr) {
+                                        if (locomotor->Is_Moving_Backwards()) {
+                                            magnitude = -magnitude;
+                                            addition = -addition;
+                                        }
+                                    }
+                                }
+
+                                float rotation = data->m_rotationSpeedMultiplier;
+                                m_frontWheelRotation += rotation * magnitude;
+
+                                if (m_isPowersliding) {
+                                    m_rearWheelRotation += (magnitude + addition) * rotation;
+                                } else {
+                                    m_rearWheelRotation += rotation * magnitude;
+                                }
+
+                                m_midFrontWheelRotation = m_frontWheelRotation;
+                                m_midRearWheelRotation = m_rearWheelRotation;
+
+                                Matrix3D m(true);
+
+                                if (m_frontLeftTireBone != 0 && info) {
+                                    m.Adjust_Z_Translation(info->m_frontLeftHeightOffset);

```

Comment:
Maybe add `m.Make_Identity();` here and remove the `true` from `Matrix3D m(true); above`

Reply:
fixed.